src/parser/statement.js

```diff
@@ -285,6 +285,7 @@ pp.parseForStatement = function (node) {
     return this.parseFor(node, null);
   }
 
+  // var, let, or const begins a regular loop
```

Comment:
I think I put the comment there to remind myself that those cases were completely out of the picture after that point in the code. I can just remove it.